### ISSUE TITLE
Please add to this list of weird preprocessor results

### ISSUE THREAD

**Issue by jeffbl** [Shared-Reality-Lab/IMAGE-server#213]
When #194 is resolved, there will be a more formal way to log issues with preprocessors giving weird results, e.g., consistently wrong tags like "embassy". For now, please use this work item to report weird preprocessor results, with one graphic per comment. Please make sure to include:
- [ ] URL of graphic
- [ ] text of what it reports (for object detection, there is a text version returned as a rendering at the bottom you can copy/paste)
- [ ] What is egregious in the reported results?
- [ ] Optional: any notes about similar issues or themes you're encountering across graphics.

**Comment by rohanakut** [comment]
https://www.istockphoto.com/photo/new-york-city-asphalt-road-on-busy-intersection-streets-with-car-traffic-at-daytime-gm1133502463-300855877

after #209 :  4 people, 2 cars, 2 trucks, a bicycle, and a traffic light.

after testing on a bigger model(yolov5s6): 4 people, 2 cars, 2 trucks, a bicycle, and a traffic light.

**Comment by jeffbl** [comment]
I am closing this as the last updates are from a year ago, and are likely no longer relevant.
